js/editor-cm.js

```diff
@@ -1,5 +1,99 @@
 var EditSession = CodeMirror.Doc;
 
+// The following code was taken from chrome dev tools to help improve codemirror
+// screen reader support for navigating around a document.
+// https://github.com/ChromeDevTools/devtools-frontend/blob/0ddf8e4898701ab4174096707346d71cc5985268/front_end/text_editor/CodeMirrorTextEditor.js#L1736
+
+
+// CodeMirror uses an offscreen <textarea> to detect input. Due to inconsistencies in the many browsers it supports,
+// it simplifies things by regularly checking if something is in the textarea, adding those characters to the document,
+// and then clearing the textarea. This breaks assistive technology that wants to read from CodeMirror, because the
+// <textarea> that they interact with is constantly empty.
+// Because we target up-to-date Chrome, we can gaurantee consistent input events. This lets us leave the current
```

Comment:
is 'we target up-to-date Chrome' accurate for Text app? the wording is confusing so I'm not sure what it's implying. Text app opens as a Chrome app, but not sure we can guarantee up to date Chrome.

Reply:
i think the comment just implies that we target chrome that isn't 10 years old so we can rely on consistent events, will update the comment to be more clear